ports/stm32/mpconfigboard_common.h

```diff
@@ -256,6 +274,8 @@
 // Windows needs a different PID to distinguish different device configurations.
 #ifndef MICROPY_HW_USB_VID
 #define MICROPY_HW_USB_VID              (0xf055)
+#define MICROPY_HW_USB_PID              (0x9802)
```

Comment:
Maybe add a comment that this is only used in the TinyUSB configuration (and eventually we'll need to change this value depending on whether MSC is enabled etc).